lib/utils/bulk_import_utils.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module DEQMTestKit
+  # Utility functions in support of the bulk import test group
+  module BulkImportUtils
+    def get_retry_or_backoff_time(wait_time, reply)
+      retry_after = -1
+      is_retry_nil = reply[:headers].find { |h| h.name == 'retry_after' }
```

Comment:
The header here is supposed to be `Retry-After`, but not sure what the implications of changing this are for the functionality of the code. I would try changing it locally to make sure it doesn't break everything.